amelia.html

```
@@ -0,0 +1,614 @@
+      </div>
+
+      <!-- Presence Audit -->
+      <h2 id="audit" class="text-2xl font-semibold mb-2 scroll-mt-20">
+        Weekly presence audit
+      </h2>
+      <p class="text-slate-400 text-sm mb-4">
+        Quick honest check — Level 2 of your affirmation system, applied to
+        Amelia. Click each row.
+      </p>
+      <div
+        id="audit"
```

HTMLHint (1.9.2)

[error] The id value [ audit ] must be unique.

(id-unique)